src/classBased/components/InputTodo.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class InputTodo extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      title: '',
+    };
+  }
+
+  onChange = (e) => {
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    const { addTodoProps } = this.props;
+    const { title } = this.state;
+    if (title.trim()) {
+      addTodoProps(title);
+      this.setState({
+        title: '',
+      });
+    } else {
+      alert('Please write item');
+    }
+  };
+
+  render() {
+    const { title } = this.state;
+    return (
+      <form onSubmit={this.handleSubmit} className="form-container">
+        <input
+          type="text"
+          className="input-text"
+          placeholder="Add Todo..."
+          value={title}
+          name="title"
+          onChange={this.onChange}
+        />
+        <button type="submit" className="input-submit">
+          Submit
+        </button>
```

Comment:
### Your [InputTodo.js](https://github.com/DJ-MrJay/react-todo-app/pull/1/files#diff-89941f230a757f229a14e17006ff835825d86cb1f366b2d35eea5e9bf67ea015) File

![image](https://user-images.githubusercontent.com/98043302/192999499-e5b75299-79c1-44a5-89b0-a9e36040437f.png)

[Optional]

-  When I click the button it doesn't work the add only works by the "enter" key so kindly fix it 😊